### ISSUE TITLE
Pingback Receive is returning 400 Bad Request

### ISSUE THREAD

**Issue by EdiWang** [EdiWang/Moonglade#191]
Blog posts cannot be pinged

**To Reproduce**
1. Create a new post from another website supports pingback send
2. Reference any post from staging/prod of moonglade
3. Check Azure Application Insights
4. See error: POST for Pingback/Index returning 400

**Expected behavior**
Pingback should be recorded

**Screenshots**
![image](https://user-images.githubusercontent.com/3304703/58708995-b9d7ed80-83eb-11e9-915e-308bc5801956.png)

**Additional context**
Maybe related to AntiforgeryToken

**Comment by EdiWang** [comment]
IgnoreAntiforgeryToken is blocking the POST call, solved in #192 

New error found:

```
2019-05-31 13:56:42.2452|RD00155DB8941C|INFO|Moonglade.Web.Controllers.PingbackController|Receiving Pingback,POST https://ediwang-web-staging.azurewebsites.net/pingback,Index,13.94.41.80
2019-05-31 13:56:42.3266|RD00155DB8941C|INFO|Edi.Blog.Pingback.PingbackReceiver|Receiving Pingback from 13.94.41.80,POST https://ediwang-web-staging.azurewebsites.net/pingback,Index,13.94.41.80
2019-05-31 13:56:42.3355|RD00155DB8941C|INFO|Edi.Blog.Pingback.PingbackReceiver|Pingback received xml: <?xml version="1.0"?>
<methodCall>
<methodName>pingback.ping</methodName>
<params>
<param><value><string>https://chelseamu.com/2019/05/31/work-the-magic/</string></value></param>
<param><value><string>https://ediwang-web-staging.azurewebsites.net/post/2019/5/24/reset-azure-blob-storage-content-type-in-net-core</string></value></param>
</params></methodCall>,POST https://ediwang-web-staging.azurewebsites.net/pingback,Index,13.94.41.80
2019-05-31 13:56:42.3753|RD00155DB8941C|INFO|Moonglade.Core.PingbackService|Pingback Attempt from 13.94.41.80 is valid,POST https://ediwang-web-staging.azurewebsites.net/pingback,Index,13.94.41.80
2019-05-31 13:56:42.3753|RD00155DB8941C|INFO|Edi.Blog.Pingback.PingbackReceiver|Processing Pingback from: https://chelseamu.com/2019/05/31/work-the-magic/ to https://ediwang-web-staging.azurewebsites.net/post/2019/5/24/reset-azure-blob-storage-content-type-in-net-core,POST https://ediwang-web-staging.azurewebsites.net/pingback,Index,13.94.41.80
2019-05-31 13:56:42.9695|RD00155DB8941C|INFO|Edi.Blog.Pingback.PingbackReceiver|ExamineSourceAsync:title: Work the magic &#8211; Chelsea Mu,POST https://ediwang-web-staging.azurewebsites.net/pingback,Index,13.94.41.80
2019-05-31 13:56:42.9695|RD00155DB8941C|INFO|Edi.Blog.Pingback.PingbackReceiver|ExamineSourceAsync:containsHtml: False,POST https://ediwang-web-staging.azurewebsites.net/pingback,Index,13.94.41.80
2019-05-31 13:56:42.9860|RD00155DB8941C|INFO|Edi.Blog.Pingback.PingbackReceiver|ExamineSourceAsync:sourceHasLink: True,POST https://ediwang-web-staging.azurewebsites.net/pingback,Index,13.94.41.80
2019-05-31 13:56:43.0485|RD00155DB8941C|WARN|Microsoft.EntityFrameworkCore.Query|The LINQ expression 'where (((Convert((([p].Slug == __response_Item_Item1_0) AndAlso (Convert([p.PostPublish]?.PubDateUtc?.GetValueOrDefault(), DateTime).Date == __response_Item_Item2_Date_1)), Nullable`1) AndAlso ?[p.PostPublish] | (?[p.PostPublish] | ([p.PostPublish]?.IsPublished == True)? == True)?) AndAlso Not(?[p.PostPublish] | (?[p.PostPublish] | ([p.PostPublish]?.IsDeleted == True)? == True)?)) == True)' could not be translated and will be evaluated locally.,POST https://ediwang-web-staging.azurewebsites.net/pingback,Index,13.94.41.80
2019-05-31 13:56:43.0485|RD00155DB8941C|WARN|Microsoft.EntityFrameworkCore.Query|The LINQ expression 'FirstOrDefault()' could not be translated and will be evaluated locally.,POST https://ediwang-web-staging.azurewebsites.net/pingback,Index,13.94.41.80
2019-05-31 13:56:44.5356|RD00155DB8941C|ERROR|Moonglade.Core.PingbackService|,POST https://ediwang-web-staging.azurewebsites.net/pingback,Index,13.94.41.80
2019-05-31 13:56:44.5356|RD00155DB8941C|INFO|Moonglade.Web.Controllers.PingbackController|Pingback Processor Error: GenericError,POST https://ediwang-web-staging.azurewebsites.net/pingback,Index,13.94.41.80
```

**Comment by EdiWang** [comment]
Root cause found: 

```csharp
public Response<(Guid Id, string Title)> GetPostIdTitle(string url);
```
is setting incorrect response status.

![image](https://user-images.githubusercontent.com/3304703/58713037-5bfbd380-83f4-11e9-9d82-727c95fa6c93.png)
